src/screens/UserPortal/Settings/Settings.tsx

```
@@ -56,179 +82,126 @@ export default function settings(): JSX.Element {
-    grade: '',
-    empStatus: '',
-    maritalStatus: '',
-    address: '',
-    state: '',
-    country: '',
-    image: '',
-    eventsAttended: [] as InterfaceEvent[],
-  });
+  const handleUpdateUserDetails = async (): Promise<void> => {
+    // takes an object and removes all the empty fields
+    function removeEmptyFields(obj: typeof userDetails) {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)